### PR TITLE
Add communication details

### PR DESCRIPTION
**Pull request by oraNod**
#### What is this PR doing:
As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR adds the communication section template defined by the community to the README and the docsite. Similar PRs are now being raised across all the Ansible ecosystem projects.

Thanks!

Issue: AAP-27579

#### Reviewers must know:
n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,29 +8,41 @@ This is a brand new take on Ansible Galaxy, so it will look and feel a bit diffe
...
 
 * If you're interested in jumping in and helping out, [view the contributing guide](https://github.com/ansible/galaxy_ng/wiki#contributing-to-galaxyng).
 * To setup your development environment, [view the development setup guide](https://github.com/ansible/galaxy_ng/wiki/Development-Setup).
-* Chat with us on irc.libera.chat: #ansible-galaxy
 * Found a bug or have a feature idea? Please [open an issue](https://issues.redhat.com/projects/AAH/issues).
 
+## Code of Conduct
+
+Please see the official
+[Ansible Community Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
```
Comment (Andersson007) [inline README.md:40]:
```suggestion
Please see the [Ansible Community Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
```
Maybe an extra thing? (same in the other PRs) If it sounds better for a native ear, feel free to just mark this comment as resolved

File: docs/community/devstack.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ As noted in the overview, GalaxyNG is considered a [pulp plugin](https://docs.pu
...
 
-As a hopeful contributor, you may need to first figure out where to write code for a new feature or a bugfix. General guidance is to start any new enhancements for api/v3 in the pulp_ansible project. Anything for api/v1 is in the galaxy_ng project. If you aren't sure or can't figure out where the code should be, please reach out on the IRC channels and we'll help guide you.
+As a hopeful contributor, you may need to first figure out where to write code for a new feature or a bugfix. General guidance is to start any new enhancements for api/v3 in the pulp_ansible project. Anything for api/v1 is in the galaxy_ng project. If you aren't sure or can't figure out where the code should be, please reach out on the Ansible forum and we'll help guide you.
```
Comment (Andersson007) [inline docs/community/devstack.md:9]:
```suggestion
As a hopeful contributor, you may need to first figure out where to write code for a new feature or a bugfix. General guidance is to start any new enhancements for api/v3 in the pulp_ansible project. Anything for api/v1 is in the galaxy_ng project. If you aren't sure or can't figure out where the code should be, please reach out on the Ansible forum and we'll help guide you.
```
Should we put a reference to the communication guide behind `the Ansible forum` here?

Comment (oraNod) [reply docs/community/devstack.md]: @Andersson007 updated. PTAL

**Review by Andersson007 — APPROVED** [review]

**Review by gundalow — COMMENTED** [review]
Bulk change from `latest` to `devel`?

**Review by gundalow — APPROVED** [review]

**Comment by Andersson007** [comment]
@oraNod while you're here, could you please grep once more `grep -r "freenode\|libera\|mailing\|groups.google" .` and remove the mentioned?
I screwed up another PR by pressing  "Update the branch" + trying to squash it

**Comment by oraNod** [comment]
> @oraNod while you're here, could you please grep once more `grep -r "freenode\|libera\|mailing\|groups.google" .` and remove the mentioned? I screwed up another PR by pressing "Update the branch" + trying to squash it

@Andersson007 as mentioned in matrix there's one reference to google groups in docs here: https://github.com/ansible/galaxy_ng/blob/8ab50e789871af804001630648920442a9585eef/docs/community/overview.md?plain=1#L15

it's an href to something of historical interest, rather than a suggested comms method. let's leave it be. cheers.

**Comment by Andersson007** [comment]
@oraNod SGTM, thanks for checking, +1 to merge the PR as-is

**Review by gundalow — APPROVED** [review]

**Pull request merged**